pkg/templates/aws/resources/load_balancer_listener.yaml

```diff
@@ -2,23 +2,43 @@ qualified_type_name: aws:load_balancer_listener
 display_name: Load Balancer Listener
 
 properties:
+  Certificate:
+    type: resource(aws:acm_certificate)
+    operational_rule:
+      if: '{{ eq (fieldValue "Protocol" .Self) "HTTPS" }}'
+      step:
+        direction: downstream
+        resources:
+          - aws:acm_certificate
   Port:
     type: int
     default_value: 80
+    min_value: 1
+    max_value: 65535
+    required: true
   Protocol:
     type: string
+    allowed_values:
+        - GENEVE
```

Comment:
still curious on what this is

Reply:
Looks like it's for gateway load balancers: https://docs.aws.amazon.com/elasticloadbalancing/latest/gateway/gateway-load-balancers.html.